engine-rest/engine-rest-openapi/src/main/templates/paths/schema-log/post.ftl

```diff
@@ -0,0 +1,45 @@
+{
+  <@lib.endpointInfo
+      id = "querySchemaLogs"
+      tag = "Schema Log"
+      desc = "Queries for schema log entries that fulfill given parameters." />
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "SchemaLogQueryDto"
+      examples = ['"example-1": {
+                         "summary": "POST /schema/log",
+                         "description": "The content of the Request Body",
+                         "value": {
+                           "version": "7.11.0",
+                           "sortBy": "timestamp",
+                           "sortOrder": "asc"
+                         }
+                   }'] />
+
+  "parameters" : [
+    <#assign last = true >
+    <#include "/lib/commons/pagination-params.ftl" >
+  ],
```

Comment:
I would suggest keeping the order of `parameters` and `requestBody` always the same, that way it will be close to docs-manual as well and easier to read, navigate.
```suggestion
  "parameters" : [
    <#assign last = true >
    <#include "/lib/commons/pagination-params.ftl" >
  ],
  
  <@lib.requestBody
      mediaType = "application/json"
      dto = "SchemaLogQueryDto"
      examples = ['"example-1": {
                         "summary": "POST /schema/log",
                         "description": "The content of the Request Body",
                         "value": {
                           "version": "7.11.0",
                           "sortBy": "timestamp",
                           "sortOrder": "asc"
                         }
                   }'] />

```

Reply:
Good to know, I was considering if I should do that. Thanks!